client/my-sites/stats/site.jsx

```diff
@@ -563,6 +564,25 @@ class StatsSite extends Component {
 							/>
 						) }
 
+						{ /* Either stacks with Clicks or with Emails depending on UTM */ }
+						{ ! this.isModuleHidden( 'authors' ) && isNewStateEnabled && (
+							<StatsModuleAuthors
+								path="authors"
```

Comment:
Looks like `path`, `showSummaryLink` and `statType` are not needed.

Reply:
yep you're right, good catch. Thanks!